src/viam/components/base/base.py

```diff
@@ -92,6 +109,34 @@ async def set_power(
         When ``linear`` and ``angular`` are both nonzero, the base will move in an arc,
         with a tighter radius if angular power is greater than linear power.
 
+        ::
+
+            my_base = Base.from_robot(robot=robot, name="my_base")
+
+            # Make your wheeled base move forward. Set linear power to 75%.
+            print("move forward")
+            await my_base.set_power(
+                linear=Vector3(x=0, y=-.75, z=0),
+                angular=Vector3(x=0, y=0, z=0))
```

Comment:
For these, you could pass in an empty `Vector3()` rather than initializing all the values to `0` to make things a bit cleaner, but initializing to `0` might be good for docs as it shows the explicit values

Reply:
I don't have a strong opinion but I do like to show it explicitly for readability as you say